resources/run_adf.sh

```diff
@@ -49,9 +49,24 @@ GO="$GOBINDIR/go"
 
 PATH=$GOBINDIR:$PATH
 
+# GITHUB_TOKEN must be set for cloning 10gen/mongohouse repository from Evergreen hosts, and the
+# dependencies needed to build it.
+# If unset, it will default to using the SSH private key on the local system.
+if [[ ${GITHUB_TOKEN} ]]; then
+  # Clear git url configurations if they exist
+  git config --global --get-regexp '^url\.' | while read -r key _; do
```

Comment:
The global git config doesn't get reset by Evergreen so this was causing intermittent failures.  I see they have a ticket to consider fixing that [here](https://jira.mongodb.org/browse/DEVPROD-10373).  This change will make it so we don't try to use an existing token configuration.  I will make this change in the jdbc repo also.

Reply:
QQ: Why is not necessary for the PowerBI connector run_adf script?

Reply:
Oh whoops, I forgot to push the commit.  It's there now in the powerbi connector. 👍